src/pages/Main/Main.js

```diff
@@ -1,6 +1,122 @@
 import React from 'react';
 import styled from 'styled-components';
 
+const MainSearchLabel = ({
+  text,
+  bgColor,
+  color,
+  textAlign,
+  fontSize,
+  ...rest
+}) => {
+  return (
+    <SearchLabel bgColor={bgColor}>
+      <SearchText>{text}</SearchText>
+      <SearchInput
+        bgColor={bgColor}
+        color={color}
+        textAlign={textAlign}
+        fontSize={fontSize}
+        {...rest}
+      ></SearchInput>
+    </SearchLabel>
+  );
+};
+
+const MainWrapper = () => {
+  return (
+    <MainWrap>
+      <MainImg
+        alt="Main Background Image"
+        src="/images/Main/MainBackGroundImg.jpg"
+        width="100%"
+        height="100%"
+      />
+      <MainSearchForm>
+        <MainSearchLabel text="지역" placeholder="어디로 여행가시나요?" />
+        <MainSearchLabel
+          text="날짜"
+          placeholder="체크인, 체크아웃"
+          type="button"
+          value="체크인, 체크아웃"
+        />
+        <MainSearchLabel
+          type="button"
+          value="검색"
+          bgColor="#74b9ff"
+          color="white"
+          textAlign="center"
+          fontSize="25px"
+        />
+      </MainSearchForm>
+    </MainWrap>
+  );
+};
+
 export const Main = () => {
-  return <div></div>;
+  return (
+    <Section>
+      <MainWrapper />
+    </Section>
+  );
 };
+
+const SearchText = styled.span`
+  margin-left: 10px;
+  margin-bottom: 5px;
+  font-size: 12px;
+`;
+
+const SearchInput = styled.input`
+  width: 180px;
+  padding: 0 10px;
+  border: none;
+  background-color: ${props => props.bgColor || 'white'};
+  color: ${props => props.color || 'black'};
+  font-size: ${props => props.fontSize || '16px'};
+  text-align: ${props => props.textAlign || 'left'};
```

Comment:
```suggestion
  background-color: ${({bgColor}) => props.bgColor || 'white'};
```
매개변수자리에서 구조분해 가능합니다.

```suggestion
  ${({bgColor, color, fontSize, textAlign}) => `
     background-color: ${bgColor || 'white'};
     color: ${color || 'black'};
     font-size: ${fontSize || '16px'};
     text-align: ${textAlign || 'left'};
  `}
```
이렇게도 표현할 수 있습니다.

Reply:
연욱님 해당 방법에서 prettier 적용은 안되는게 맞나요?